database/mainProduct.js

```diff
@@ -21,7 +21,6 @@ const mainProductSchema = new mongoose.Schema({
 
 const mainProduct = mongoose.model('mainProduct', mainProductSchema);
 
-
 // photos Schema 
 
 const photosSchema = new mongoose.Schema({
```

Comment:
I think every photo has only one product it's associated with so product_id probably doesn't need to be an array